pallets/pallet-web3-names/src/lib.rs

```diff
@@ -330,6 +330,28 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Transfer the storage deposit from one account to another.
+		///
+		/// If the currently required deposit is different, the new deposit
+		/// value will be reserved.
```

Comment:
S: Add requirements about origin.

Reply:
I wasn't sure if we want to mention DIDs since it could be anything. But we probably just want to say dids instead of subject etc? Especially since these comments might be exposed to users when they sign transactions? So I might rework the comments again. Maybe we need to rework most of the comments to be more readable for users?

[95795e9](https://github.com/KILTprotocol/kilt-node/pull/405/commits/95795e94fe3a3938ba05d6af488144803575663e)